docs/system-requirements.md

```diff
@@ -0,0 +1,112 @@
+| Redis                    | **256 MB – 2 GB**             | Session storage and caching; scales with active user sessions                          |
+| Object Storage (optional)| **External or self-hosted**   | Can use AWS S3, Azure Blob, Google Cloud Storage, or self-hosted MinIO               |
+| Django API (+ Celery)    | **1 – 3 GB**                 | Production workloads with background tasks and higher concurrency                      |
+| Static Files (Nginx)     | **< 200 MB**                 | Serves Next.js build output and static assets; no development overhead                |
+| Y-Provider (y-websocket) | **200 MB – 1 GB**             | Scales with concurrent document editing sessions                                        |
+| Nginx (Load Balancer)    | **< 200 MB**                  | Reverse proxy, SSL termination, static file serving                                    |
+
+### Production Architecture Notes
+
+- **Frontend**: Uses pre-built Next.js static assets served by Nginx (no Node.js runtime needed)
+- **Authentication**: Any OIDC-compatible provider can be used instead of self-hosted Keycloak
+- **Object Storage**: External services (S3, Azure Blob) or self-hosted solutions (MinIO) are both viable
+- **Database**: Consider PostgreSQL clustering or managed database services for high availability
+- **Scaling**: Horizontal scaling is recommended for Django API and Y-Provider services
+
+### Minimal Production Setup (Core Services Only)
+
+| Service                  | Memory    | Notes                                   |
+| ------------------------ | --------- | --------------------------------------- |
+| PostgreSQL               | **2 GB**  | Core database                           |
+| Django API (+ Celery)    | **1.5 GB**| Backend services                        |
+| Y-Provider               | **200 MB**| Real-time collaboration                 |
+| Nginx                    | **100 MB**| Static files + reverse proxy           |
+| Redis                    | **256 MB**| Session storage                         |
+| **Total (without auth/storage)** | **≈ 4 GB** | External OIDC + object storage assumed |
+
+## 4. Recommended Software Versions
+
+| Tool                    | Minimum |
+| ----------------------- | ------- |
+| Docker Engine / Desktop | 24.0    |
+| Docker Compose          | v2      |
+| Git                     | 2.40    |
+| **Node.js**             | 22+     |
+| **Python**              | 3.12+   |
+| GNU Make                | 4.4     |
+| Kind                    | 0.22    |
+| Helm                    | 3.14    |
+| kubectl                 | 1.29    |
+| mkcert                  | 1.4     |
+
+[1]: https://nodejs.org/en/about/previous-releases "Node.js — Node.js Releases"
+[2]: https://www.python.org/downloads/release/python-3120/ "Python Release Python 3.12.0 | Python.org"
+
```

Comment:
links seems not used and and identifiers `1` and `2` are already used in this page.